anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
That's a good question.  I know rc's have only recently been introduced.  How do they fit into the schedule and planning here?  If an rc tag should skip this prompt, then we can make it so.  As it works now, the prompt will only happen if X.Y-1.0 doesn't yet exist, but the user can still override.